set.mm

```diff
@@ -32039,7 +32039,7 @@ his New Foundations set theory (axiom system NF of [Quine] p. 331).  In
        ~ elirrv (derived from the Axiom of Regularity), so for us the Russell
        class equals the universe ` _V ` (theorem ~ ruv ).  See ~ ruALT for an
        alternate proof of ~ ru derived from that fact.  (Contributed by NM,
-       7-Aug-1994.) $)
+       7-Aug-1994.)  (Proof modification is discouraged.) $)
```

Comment:
@digama0 : do we really want to add the discouragement tag ? (if yes, we should provide a short reason, to write just before the tag)